website/content/docs/secrets/pki/est.mdx

```diff
@@ -0,0 +1,181 @@
+---
+layout: docs
+page_title: Enrollment over Secure Transport (EST) within Vault | PKI - Secrets Engines
+description: An overview of the Enrollment over Secure Transport protocol implementation within Vault.
+---
+
+# PKI secrets engine - Enrollment over Secure Transport (EST) <EnterpriseAlert inline="true" />
+
+@include 'alerts/beta.mdx'
+
+This document covers configuration and limitations of Vault's PKI Secrets Engine
+implementation of the [EST protocol](https://datatracker.ietf.org/doc/html/rfc7030) <EnterpriseAlert inline="true" />.
+
+## What is Enrollment over Secure Transport (EST)?
+
+The EST protocol is an IETF standardized protocol, [RFC 7030](https://datatracker.ietf.org/doc/html/rfc7030),
+that allows clients to acquire client certificates and associated
+Certificate Authority (CA) certificates.
+
+## Enabling EST support on a Vault PKI mount
+
+The following is a list of steps required to configure an existing PKI
+mount to serve EST clients. Each of which can be broken down within three main
+categories.
+
+ 1. [Authentication mechanisms](#configuring-est-authentication)
+ 1. [Updating PKI tunable parameters](#updating-the-pki-mount-tunable-parameters)
+ 1. [PKI EST configuration](#pki-est-configuration)
+
+### Configuring EST Authentication
+
+The EST protocol specifies a few different authentication mechanisms, of which
+Vault supports two.
+
+ 1. [HTTP-Based Client authentication](/vault/docs/auth/userpass)
+ 1. [Certificate TLS authentication](/vault/docs/auth/cert)
+
+Both of these authentication mechanisms leverage a separate Vault authentication
+mount, within the same namespace, to validate the client provided credentials
+along with the client's ACL policy to enforce. While both authentication
+schemes can be enabled at once, only a single mount will be used to authenticate
+a client based on the way credentials were provided through EST. If an EST client sends
+HTTP-Based authentication credentials, they will be preferred over TLS client
+certificates.
+
+When setting up the authentication mount for EST clients, the token type must
+be configured to return [batch tokens](/vault/docs/concepts/tokens#batch-tokens).
+Batch tokens are required to avoid lease explosions as every EST incoming
```

Comment:
How about? I don't believe memory overhead is the only concern about having a large number of leases.

```
Batch tokens are required to avoid an excessive amount of leases being generated
and persisted as every EST incoming request needs to be authenticated.
```